src/js/media-seek-backward-button.js

```diff
@@ -39,7 +39,11 @@ class MediaSeekBackwardButton extends MediaChromeButton {
 
   constructor(options = {}) {
     super({ slotTemplate, ...options });
+  }
+
+  connectedCallback(){
     this.seekOffset = getNumericAttr(this, Attributes.SEEK_OFFSET, DEFAULT_SEEK_OFFSET);
+    super.connectedCallback();
```

Comment:
it depends on use case but I think generally the super calls go before the code in the extending class.

`this.seekOffset` could potentially be changed in `super.connectedCallback()`